index.html

```diff
@@ -1,6 +1,6 @@
 <title> Synapse Documentation </title>
-<meta http-equiv="Refresh" content="0; url='https://matrix-org.github.io/synapse/develop/welcome_and_overview.html'" />
+<meta http-equiv="Refresh" content="0; url='https://matrix-org.github.io/synapse/develop" />
```

Comment:
Note that GitHub Pages will serve `develop/index.html` if we navigate to `develop/`.